pyproject.toml

```
@@ -38,8 +39,8 @@ sudachi = ["sudachipy", "sudachidict-core"]
 sentencepiece = ["sentencepiece"]
```

```suggestion
nagisa = ["nagisa"]
sentencepiece = ["sentencepiece"]
```

Please create a section for `nagisa`, which enables users to install konoha only with nagisa (`pip install konoha[nagisa]`).

---

Thank you for the comment. I added the row.

---

It seems that I have no valid credentials for S3 in CI. How should I do ... ?

---

You're right. For testing features of remote resources, I embed some credentials in my repo and use [here](https://github.com/himkt/konoha/blob/master/.github/workflows/pythonpackage.yml#L75-L76).
These are not accessible from forked one, then the test fails.

I'll skip tests that need credentials if they are not provided in another PR.
Anyway, your work is solid and please don't worry about it!